packages/central-server/src/database/utilities/assertCanAddDataElementInGroup.js

```diff
@@ -3,11 +3,23 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 
-const areBothDefinedAndDifferent = (oldValue, newValue) =>
-  oldValue !== undefined && newValue !== undefined && oldValue !== newValue;
+const areBothDefinedAndDifferent = (a, b) => a !== undefined && b !== undefined && a !== b;
 
-const constructErrorMessage = ({ property, value, dataElementCode, otherGroupCode }) =>
-  `Cannot set ${property} to '${value}': data element '${dataElementCode}' is included in data group '${otherGroupCode}', which uses another ${property}`;
+const areDifferentAllowingNull = (a, b) => {
+  if (a === null && b === null) return false;
+  if (a === null && b !== null) return true;
+  if (a !== null && b === null) return true;
+  return a !== b;
+};
+
+const constructErrorMessage = ({
```

Comment:
I updated these argument names and the error message for clarity, functionality is the same